src/ProjectManager.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Scanner;
+
+public class ProjectManager extends Employee {
+	protected int pmExperience;
```

Comment:
add some validation here

Reply:
I added the validation for this attribute.